<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Push Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Periodic Notification</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Begin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTimeEdit" name="timeEdit_begin">
         <property name="time">
          <time>
           <hour>9</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_add_per_notif">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_remove_per_notif">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTimeEdit" name="timeEdit_period">
         <property name="time">
          <time>
           <hour>0</hour>
           <minute>1</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="timeEdit_end">
         <property name="time">
          <time>
           <hour>17</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_per_text">
         <property name="text">
          <string>periodic notification 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QTreeWidget" name="treeWidget_per_notifs">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Everyday Notification</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QTimeEdit" name="timeEdit_daytime"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_add_day_notif">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_remove_day_notif">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_day_text">
         <property name="text">
          <string>everyday notification 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QTreeWidget" name="treeWidget_day_notifs">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Clients</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Client ip:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_ip"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_add_client">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_remove_client">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTreeWidget" name="treeWidget_clients">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
